### PR TITLE
Release 0.6

### PR DESCRIPTION
**Pull request by christianbumann**

### CODE REVIEW

File: src/Cake.Jira/Cake.Jira.csproj [hunk 1]
```diff
@@ -6,9 +6,9 @@
     <Product>Cake.Jira</Product>
     <Description>Cake build addin for integration with Jira.</Description>
     <Copyright>Copyright © Hugo Tavares 2018</Copyright>
-    <Version>0.5.0</Version>
-    <AssemblyVersion>0.5.0.0</AssemblyVersion>
-    <FileVersion>0.5.0.0</FileVersion>
+    <Version>0.6.0</Version>
```
Comment (Ninglin) [inline src/Cake.Jira/Cake.Jira.csproj]: This change should be removed as this is managed by the pipeline.

File: appveyor.yml [hunk 2]
```diff
@@ -1,9 +1,20 @@
...
   do_not_increment_build_number: true
 skip_tags: true
 environment:
   Verbosity: Normal
+before_build:
+  # Display .NET Core version
+  - cmd: dotnet --version
+  # Display minimal restore text
+  - cmd: dotnet restore ./src/Cake.Jira/Cake.Jira.csproj --verbosity m
+  - cmd: dotnet restore ./src/Cake.Jira.Tests/Cake.Jira.Tests.csproj --verbosity m
```
Comment (Ninglin) [inline appveyor.yml]: I'm not seeing this removed in your last commit.

Comment (christianbumann) [reply appveyor.yml]: Sorry my mistake wrong fetching

**Comment by christianbumann** [comment]
@Ninglin Removing the release number commit ends in an exception on [AppVeyor](https://ci.appveyor.com/project/Ninglin/cake-jira/builds/20792707). Shouldn't this be incremented by AppVeyor and configured in the [GitVersion.yml](https://github.com/Ninglin/Cake.Jira/blob/master/GitVersion.yml)?

**Comment by Ninglin** [comment]
@christianbumann that process has never been properly tested since this is a new repo. I'm checking the build locally. The problem is the version generated by Gitversion and trying to set the Appveyor bbuild number to that version. Apparently it has nothing to do with your changes.

**Pull request merged**